### PR TITLE
Warn users about exposing their emails with (B)CCs

### PR DESCRIPTION
**Pull request by Vinnl**
# New feature description

This adds a warning message to forwarded emails for Premium users (because they can reply to emails) warning them that adding back CCs and BCCs will expose their real email address.

# How to test

~~**Question:** How do I deploy to `dev` before merging to `main`? I haven't been able to test this myself. (The procedure should be: send an email to a Premium account's alias, and verify that it shows the warning in the header and looks OK.)~~

I wrote instructions in the UI Engineering onboarding doc. This is what it looked like for me:

Screen Shot 2021-12-06 at 18 32 52

# Checklist

- [x] l10n dependencies have been merged, if any. https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/47 is merged.
- [x] All acceptance criteria are met.
- [x] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/static/scss/libs/protocol/css/includes/tokens/dist/index.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: emails/templates/emails/wrapped_email.html [hunk 1]
```diff
@@ -45,6 +45,11 @@
                   {% bold_violet_text display_email as bolded_email %}
                   {% bold_violet_link SITE_ORIGIN 'Firefox Private Relay' as linked_origin %}
                   {% ftlmsg 'forwarded-email-header-from' display_email=bolded_email linked_origin=linked_origin as email_wrap_forwarded_from %}
+                  {% if recipient_profile.has_premium %}
```
Comment (Vinnl) [inline emails/templates/emails/wrapped_email.html:48]: Question: is `recipient_profile` the same as regular `profile`s? In other words, can I access this property on there? (/cc @groovecoder)

Comment (Vinnl) [reply emails/templates/emails/wrapped_email.html]: Confirmed that it is!

File: emails/templates/emails/wrapped_email.html [hunk 2]
```diff
@@ -57,6 +57,11 @@
                     <br><a href="{{ survey_link }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">{{ survey_text }}</a>.
                     {% endif %}
                   {% endif %}
+                  {% if recipient_profile.has_premium %}
+                    <br />
+                    <img alt="" width="24" style="display: inline-block; margin-bottom: 0px; width: 24px;"  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAACXBIWXMAAA7DAAAOwwHHb6hkAAAA GXRFWHRTb2Z0d2FyZQB3d3cuaW5rc2NhcGUub3Jnm+48GgAAAX1JREFUSIntlLEvQ2EUxX/3a+uN JcZuXRm6EIn4AyTWSqh0a5sOIozCPyFEWhual7B2EQtRQ1kk5tpsTS0MbT3X4BHE1+cZMPRMN985 95x77/BBH38NCRJsTKvjDJNDWQBGARSuRagMtSnPHkrnxwHbWU0YjyqQskiuniLMFHfl1uZhek3+ Zi40RJmLeiSiHgkV5oEbIGU8qgdpHbD5RG2Ef5YUQgOPibwrzXe0W5rTYwx1IHXnkAc2Q23g3xyF 9cJHcwAKrjQVVgFUydhs7AEwAhB75NQmcGIcAYivDRvwANCNMGgTdNsk/PL+JwEnAEbI2gRqXjjx taECjLAFoMrK9ryOf+b9t2VAPV60oQJy+3KmsAPEjOCW0hp/5UppjRvBBWJAuViR89ABAJ0Wi8Al kFSHtdd3v04KXLRbLPXyCPwqyhkdU7iwjDdR2JN6r/6eGwDkK3KJUPtitFqQ+a8g8EQA5YzWFCY/ ddYK+zIV1Bt4IgAV9Du6Pv4nngE8HXRqmKzo6wAAAABJRU5ErkJggg==" />
```
Comment (groovecoder) [inline emails/templates/emails/wrapped_email.html:62]: Follow-up: should we add `alt` text here for accessibility?

**Review by groovecoder — APPROVED** [review]
Code and spot-check looks good.

File: emails/templates/emails/wrapped_email.html (hunk 2 above)
Comment (Vinnl) [reply emails/templates/emails/wrapped_email.html:62]: The `i` icon doesn't convey any extra information, so an empty alt text is appropriate here IMHO.

**Pull request merged**